autorun.brs

```diff
@@ -291,16 +280,19 @@ Sub EnterEventLoop()
           receivedLoadFinished = true
         else if eventData.reason = "message" then
           DebugLog("BS: Message receved: " + FormatJson(eventData.message, 0))
-          if eventData.message.screenshotInterval <> invalid then
-            gaa.screenshotTimer.SetElapsed((val(eventData.message.screenshotInterval) / 1000), 0)
```

Comment:
screenshot interval was changed only for the current session and it wasn't persistent in some kind of settings. This means that after reboot a player was reverting back to the default interval.